airflow-pipeline/build.gradle

```diff
@@ -0,0 +1,46 @@
+plugins {
+    id 'com.github.johnrengelman.shadow' version '6.0.0'
+
+    // Apply the scala plugin to add support for Scala
+    id 'scala'
+
+    id 'idea'
+
+    id 'application'
+}
+
+ext.scala_version = "2.12.1"
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    // For running just with Spark Core
+    //implementation 'org.apache.spark:spark-core_2.12:3.0.0'
+    implementation 'org.scala-lang:scala-library:2.12.1'
+
+    // Spark SQL subsumes Spark Core
+    implementation 'org.apache.spark:spark-sql_2.12:3.0.0'
+
+    implementation group: 'com.github.pureconfig', name: 'pureconfig_2.12', version: '0.17.1'
```

Comment:
I believe there's a function like `scalaSuffix` in order to avoid hardcoding scala major version.

Reply:
added ext variables